cmd/README.mysql.md

```diff
@@ -0,0 +1,268 @@
+# HarbourBridge: Turnkey MySQL-to-Spanner Evaluation
+
+HarbourBridge is a stand-alone open source tool for Cloud Spanner evaluation,
+using data from an existing PostgreSQL or MySQL database. This README provides
+details of the tool's MySQL capabilities. For general HarbourBridge information
+see this [README](https://github.com/cloudspannerecosystem/harbourbridge#harbourbridge-turnkey-spanner-evaluation).
+
+## Example MySQL Usage
+
+The following examples assume `harbourbridge` has been added to your PATH
+environment variable.
+
+HarbourBridge can either be used with mysqldump or it can be run directly
+on a MySQL database (via go's database/sql package).
+
+### Using HarbourBridge with mysqldump
+
+To use HarbourBridge on a MySQL database called mydb using mysqldump, run:
+
+```sh
+mysqldump mydb | harbourbridge schema -source=mysql
+```
+
+The tool can also be applied to an existing mysqldump file:
+
+```sh
+harbourbridge schema -source=mysql < my_mysqldump_file
+```
+
+To specify a particular Spanner instance to use, run:
+
+```sh
+harbourbridge schema -source=mysql -target-profile="instance=my-spanner-instance" < my_mysqldump_file
```

Comment:
Should we still keep the instance flag in schema mode?

Reply:
Good catch, updated these sample commands to use data subcommand instead.